src/analysis/normed_space/exponential.lean

```diff
@@ -421,18 +432,14 @@ begin
   exact ring.inverse_invertible _,
 end
 
-lemma commute.exp {x y : 𝔸} (h : commute x y) :
```

Comment:
It might be worth keeping even if the proof is simply `commute.exp_left` followed by `commute.exp_right`? That looks like a useful shortcut.

Reply:
Sure, happy to add it back when a cache is available. I couldn't find `commute.neg` so assumed we didn't care about that type of shortcut; but I guess it's convenient here just to handle the `K` argument.